ui/src/components/EntitySearch.vue

```diff
@@ -100,7 +100,8 @@ export default {
       if (this.existingEntities) {
         return this.entities.filter(ent => {
           for (let otherEnt of this.existingEntities) {
-            if (ent[this.idField] === otherEnt[this.idField]) {
+            // using double equal to convert string to numbers
```

Comment:
Prefer `===` and use either a unary `+` or `parseInt` to convert to number.

Reply:
Okay. I see!